sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1222,169 +1030,1990 @@
-      "tableView": false,
-      "modalEdit": false,
-      "values": [
-        {
-          "label": "Yes",
-          "value": "yes",
-          "shortcut": ""
-        },
-        {
-          "label": "No",
-          "value": "no",
-          "shortcut": ""
-        }
-      ],
-      "dataType": "",
-      "persistent": true,
-      "protected": false,
       "dbIndex": false,
-      "encrypted": false,
-      "redrawOn": "",
-      "clearOnHide": true,
       "customDefaultValue": "",
       "calculateValue": "",
       "calculateServer": false,
-      "allowCalculateOverride": false,
+      "widget": null,
+      "validateOn": "change",
       "validate": {
-        "required": true,
-        "customMessage": "",
+        "required": false,
         "custom": "",
         "customPrivate": false,
-        "json": "",
         "strictDateValidation": false,
         "multiple": false,
         "unique": false
       },
-      "errorLabel": "",
-      "key": "deliveredOnlineAlsoOnsite",
-      "tags": [],
-      "properties": {},
-      "conditional": {
-        "show": true,
-        "when": "programDeliveryTypes",
-        "eq": "deliveredOnline",
-        "json": ""
-      },
-      "customConditional": "",
-      "logic": [],
-      "attributes": {},
-      "overlay": {
-        "style": "",
-        "page": "",
-        "left": "",
-        "top": "",
-        "width": "",
-        "height": ""
-      },
-      "type": "radio",
-      "input": true,
-      "placeholder": "",
-      "prefix": "",
-      "suffix": "",
-      "multiple": false,
-      "unique": false,
-      "refreshOn": "",
-      "widget": null,
-      "validateOn": "change",
+      "allowCalculateOverride": false,
+      "encrypted": false,
       "showCharCount": false,
       "showWordCount": false,
       "allowMultipleMasks": false,
-      "inputType": "radio",
-      "fieldSet": false,
-      "id": "e50a9zh",
-      "defaultValue": "",
-      "dataGridLabel": false
-    },
-    {
-      "label": "Will the students earn the same number of credits int he same time period as students in other StudentAid BC eligible programs delivered on site?",
-      "labelPosition": "top",
-      "optionsLabelPosition": "right",
-      "description": "",
-      "tooltip": "",
-      "customClass": "",
-      "tabindex": "",
-      "inline": false,
-      "hidden": false,
-      "hideLabel": false,
-      "autofocus": false,
-      "disabled": false,
-      "tableView": false,
-      "modalEdit": false,
-      "values": [
+      "tree": false,
+      "components": [
+        {
+          "label": "Are students able to take this on a part time basis?",
+          "labelPosition": "top",
+          "optionsLabelPosition": "right",
+          "description": "",
+          "tooltip": " A part-time program has a course load between 20 and 59%. A full-time program must have a course load of: 60% or greater or Between 40 and 60% for students with a permanent disability",

```

Comment:
Please remove the white space and it seems that there is a weird character after "59%".

Reply:
There was a junk character when I copied the content from Zenhub. It has been removed now.